alleycats-core/src/main/scala/alleycats/std/iterable.scala

```diff
@@ -65,6 +65,18 @@ trait IterableInstances {
       override def traverse[G[_], A, B](fa: Iterable[A])(f: A => G[B])(implicit G: Applicative[G]): G[Iterable[B]] =
         if (fa.isEmpty) G.pure(Iterable.empty)
         else G.map(Chain.traverseViaChain(toImIndexedSeq(fa))(f))(_.toVector)
+
+      override def mapAccumulate[S, A, B](init: S, fa: Iterable[A])(f: (S, A) => (S, B)): (S, Iterable[B]) = {
```

Comment:
we can't use the StaticMethod implementation here because we aren't in the cats package.

Reply:
There is also no guarantee that the `Iterable` would be strict.

Reply:
good point.